### DISCUSSION TITLE
How do I select the vs toolchain when installing a package?

### DISCUSSION THREAD

**Discussion by pisiiki** [microsoft/vcpkg#19455] [Q&A]
I have been trying today to build v142 and v143 versions of some packages without success. Is there a simple way to handle this? ie: via parameters to vcpkg install / cmake?

**Comment by Neumann-A** [comment]
Please read: https://vcpkg.io/en/docs/users/triplets.html (`VCPKG_PLATFORM_TOOLSET` / `VCPKG_VISUAL_STUDIO_PATH`)

**Comment by PhoebeHui** [comment]
@pisiiki, we have PR https://github.com/microsoft/vcpkg/pull/19088/ and https://github.com/microsoft/vcpkg/pull/18955 to fully support VS2022. 

As a workaround, could you add 'set(VCPKG_PLATFORM_TOOLSET v143)' to your triplet file and try again?

**Comment by pisiiki** [comment]
Thanks.
